azure/requests.yaml

```diff
@@ -4,7 +4,7 @@ releases:
   - name: coredns
     version: ">= 1.14.2"
   - name: net-exporter
-    version: ">= 1.16.1"
+    version: ">= 1.16.2"
```

Comment:
unfortunately there is already an azure release > 19.0.1.
Please change the `--name` value at line 2 with something like:

```
- name "> 19.0.1 <19.999.999"
```

Reply:
thanks. it's also complaining about v20-beta1. I'll try fixing it after lunch

Reply:
that is what I was talking about.